README.md

```diff
@@ -21,6 +21,11 @@ mkdir build; cd build; cmake .. -DUSE_AVX_INSTRUCTIONS=1; cmake --build .
 
 Doing so will make some things run faster.
 
+Alternatively, if you are using the [vcpkg](https://github.com/Microsoft/vcpkg/) dependency manager you can download and install dlib with CMake integration in a single command:
+```bash
+vcpkg install dlib
+```
+
```

Comment:
Thanks for the PR.  Isn't the next paragraph somewhat contrary to this?  If someone is using vcpkg they are almost certainly using visual studio correct?  They would then not run cmake as the next paragraph indicates?  This should be clarified and this new text needs to be coherent with the surrounding text. At least by making it clear that this is an either or thing.  Either cmake, or alternatively vcpkg when using visual studio.

Reply:
@davisking, thanks for your reviewing!

VCPKG support to build dlib on Windows, Linux ans OSX. On windows, it use visual studio to build dlib, so if someone using vcpkg, it's not only using visual studio, but also gcc or clang, I considered to add it separately, could you give me some guidelines that where should I add the instructions?